deps/wazuh_testing/wazuh_testing/logcollector.py

```diff
@@ -199,3 +206,64 @@ def callback_running_command(log_format, command, prefix=monitoring.LOG_COLLECTO
     log_format_message = 'full command' if log_format == 'full_command' else 'command'
     msg = fr"DEBUG: Running {log_format_message} '{command}'"
     return monitoring.make_callback(pattern=msg, prefix=prefix, escape=escape)
+
+
+def callback_match_pattern_file(file_pattern, file):
+    """Create a callback to detect if logcollector is monitoring a file with wildcard.
+    Args:
+        file_pattern (str): Location pattern that the has to match.
+        file (str): Name with absolute path of the analyzed file.
+        prefix (str): Daemon that generates the error log.
```

Comment:
```suggestion

```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/commit/e64facee7f700035889c107f932edc537449fec2